packages/react-native-renderer/src/__tests__/ReactFabricHostComponent-test.internal.js

```diff
@@ -200,6 +200,33 @@ describe('measureLayout', () => {
   });
 });
 
+describe('unstable_getBoundingClientRect', () => {
+  test('component.unstable_getBoundingClientRect() returns DOMRect', () => {
+    const [[fooRef]] = mockRenderKeys([['foo']]);
+
+    const rect = fooRef.unstable_getBoundingClientRect();
+
+    expect(nativeFabricUIManager.getBoundingClientRect).toHaveBeenCalledTimes(
+      1,
+    );
+    expect(rect.toJSON()).toMatchObject({
+      x: 10,
+      y: 10,
+      width: 100,
+      height: 100,
+    });
```

Comment:
Note that on web this is expected to have top/left/right/bottom values too: https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect